components/Form/Input/Input.tsx

```diff
@@ -34,21 +33,31 @@ function Input({
   const isLabelBeforeInput = !isLabelAfterInput;
 
   return (
-    <div className={cx(className, styles.field)} data-testid={INPUT}>
+    <div className={cx('m-4', className)} data-testid={INPUT}>
       {isLabelBeforeInput && (
         <Label for={name} isHidden={isLabelHidden}>
           {label}
         </Label>
       )}
 
-      <div className={styles.inputFeedbackGrouping} data-testid={INPUT_FEEDBACK_GROUPING}>
+      <div
+        className="flex flex-col items-stretch lg:relative"
+        data-testid={INPUT_FEEDBACK_GROUPING}
+      >
         <input
           {...field}
           {...props}
-          className={cx(styles.Input, hasValidationStyling, {
-            [styles.valid]: touched[name] && !hasErrors && hasValidationStyling,
-            [styles.invalid]: touched[name] && hasErrors && hasValidationStyling,
-          })}
+          className={cx(
+            'border border-secondary/50 rounded-[3px] text-lg p-2',
```

Comment:
you can always remove arbitrary values like this unless you think it makes a huge difference
```suggestion
            'border border-secondary/50 rounded-sm text-lg p-2',
```